src/utils/web3.js

```diff
@@ -425,7 +425,7 @@ export const getWalletProvider = (_provider, rpcUrls) => {
 export const getProviderGasPrice = (rpcUrls) => {
   // const provider = getWalletProvider(null, rpcUrls) // TEMP DISABLE
   // const gasPrice = provider.getGasPrice() // TEMP DISABLE
-  const gasPrice = '6000000000' // TEMP OVERRIDE
+  const gasPrice = getNetwork().chainId === 97 ? '10000000000' : '6000000000' // TEMP OVERRIDE
```

Comment:
Awesome! Thanks, buddy! This will make testnet a lot less frustrating to deal with 😂
I'll make one small change to the 6 GWEI used for mainnet while we are here too as 5 GWEI works consistently now